src/set_theory/game/pgame.lean

```diff
@@ -899,58 +899,67 @@ def relabelling.neg_congr : ∀ {x y : pgame}, x.relabelling y → (-x).relabell
+    { exact or.inl ⟨w, neg_le_neg_iff.1 h⟩ } },
   { rcases h.left j with ⟨w, h⟩ | ⟨w, h⟩,
-    { exact or.inr ⟨w, neg_le_iff.1 h⟩ },
-    { refine or.inl ⟨to_right_moves_neg.symm w, neg_le_iff.1 _⟩,
+    { exact or.inr ⟨w, neg_le_neg_iff.1 h⟩ },
+    { refine or.inl ⟨to_right_moves_neg.symm w, neg_le_neg_iff.1 _⟩,
       rwa [move_left_neg_symm, neg_neg] } },
   { rcases h.right i with ⟨w, h⟩ | ⟨w, h⟩,
     { refine or.inr ⟨to_right_moves_neg w, _⟩,
-      convert neg_le_iff.2 h,
+      convert neg_le_neg_iff.2 h,
       rw move_right_neg },
-    { exact or.inl ⟨w, neg_le_iff.2 h⟩ } },
+    { exact or.inl ⟨w, neg_le_neg_iff.2 h⟩ } },
   { rcases h.left j with ⟨w, h⟩ | ⟨w, h⟩,
-    { exact or.inr ⟨w, neg_le_iff.2 h⟩ },
+    { exact or.inr ⟨w, neg_le_neg_iff.2 h⟩ },
     { refine or.inl ⟨to_left_moves_neg w, _⟩,
-      convert neg_le_iff.2 h,
+      convert neg_le_neg_iff.2 h,
       rw move_left_neg } }
 end
 using_well_founded { dec_tac := pgame_wf_tac }
 
-theorem neg_congr {x y : pgame} (h : x ≈ y) : -x ≈ -y :=
-⟨neg_le_iff.2 h.2, neg_le_iff.2 h.1⟩
+@[simp] theorem neg_lf_neg_iff {x y : pgame} : -y ⧏ -x ↔ x ⧏ y :=
+by rw [←pgame.not_le, ←pgame.not_le, not_iff_not, neg_le_neg_iff]
+
+@[simp] theorem neg_lt_neg_iff {x y : pgame} : -y < -x ↔ x < y :=
+by rw [lt_iff_le_and_lf, lt_iff_le_and_lf, neg_le_neg_iff, neg_lf_neg_iff]
+
+theorem neg_le_iff {x y : pgame} : -y ≤ x ↔ -x ≤ y :=
+by rw [←neg_neg x, neg_le_neg_iff, neg_neg]
 
-@[simp] theorem neg_lf_iff {x y : pgame} : -y ⧏ -x ↔ x ⧏ y :=
-by rw [←pgame.not_le, ←pgame.not_le, not_iff_not, neg_le_iff]
+theorem neg_lf_iff {x y : pgame} : -y ⧏ x ↔ -x ⧏ y :=
+by rw [←neg_neg x, neg_lf_neg_iff, neg_neg]
 
-@[simp] theorem neg_lt_iff {x y : pgame} : -y < -x ↔ x < y :=
-by rw [lt_iff_le_and_lf, lt_iff_le_and_lf, neg_le_iff, neg_lf_iff]
+theorem neg_lt_iff {x y : pgame} : -y < x ↔ -x < y :=
+by rw [←neg_neg x, neg_lt_neg_iff, neg_neg]
 
 @[simp] theorem neg_le_zero_iff {x : pgame} : -x ≤ 0 ↔ 0 ≤ x :=
-by { convert neg_le_iff, rw pgame.neg_zero }
+by { convert neg_le_neg_iff, rw pgame.neg_zero }
 
 @[simp] theorem zero_le_neg_iff {x : pgame} : 0 ≤ -x ↔ x ≤ 0 :=
-by { convert neg_le_iff, rw pgame.neg_zero }
+by { convert neg_le_neg_iff, rw pgame.neg_zero }
 
 @[simp] theorem neg_lf_zero_iff {x : pgame} : -x ⧏ 0 ↔ 0 ⧏ x :=
-by { convert neg_lf_iff, rw pgame.neg_zero }
+by { convert neg_lf_neg_iff, rw pgame.neg_zero }
 
 @[simp] theorem zero_lf_neg_iff {x : pgame} : 0 ⧏ -x ↔ x ⧏ 0 :=
-by { convert neg_lf_iff, rw pgame.neg_zero }
+by { convert neg_lf_neg_iff, rw pgame.neg_zero }
 
 @[simp] theorem neg_lt_zero_iff {x : pgame} : -x < 0 ↔ 0 < x :=
-by { convert neg_lt_iff, rw pgame.neg_zero }
+by { convert neg_lt_neg_iff, rw pgame.neg_zero }
 
 @[simp] theorem zero_lt_neg_iff {x : pgame} : 0 < -x ↔ x < 0 :=
-by { convert neg_lt_iff, rw pgame.neg_zero }
+by { convert neg_lt_neg_iff, rw pgame.neg_zero }
+
+theorem neg_congr {x y : pgame} (h : x ≈ y) : -x ≈ -y :=
+⟨neg_le_neg_iff.2 h.2, neg_le_neg_iff.2 h.1⟩
```

Comment:
Seems you missed a `neg_equiv_neg_iff`: https://github.com/leanprover-community/mathlib/blob/dfa2eb5cbcefc22fa2c5b795f91771bf368ea2bb/src/set_theory/game/pgame.lean#L943-L946

Reply:
I've added `neg_equiv_neg_iff` and `neg_equiv_iff`.

Reply:
My `neg_equiv_iff` name doesn't conform to the convention should refer to `-x ≈ y ↔ -y ≈ x` instead.

Would you also add the fuzzy versions? And maybe the `*_neg_iff` forms, like `y < -x`? Or maybe we should generalize lemmas in [algebra.order.group](https://leanprover-community.github.io/mathlib_docs/algebra/order/group.html) to use `has_involutive_neg` instead of `add_group` (does that work)?

Reply:
Sure, I'll add the fuzzy version.

I don't think that works, we would rather need some sort of typeclass for `a - a = 0`, plus some covariant classes.